File: src/main/java/io/github/dqualizer/dqapi/repositories/rqa/RqaDefinitionRepository.java
package io.github.dqualizer.dqapi.repositories.rqa;

import io.github.dqualizer.dqlang.types.rqa.definition.RuntimeQualityAnalysisDefinition;
import org.springframework.data.mongodb.repository.MongoRepository;


public interface RqaDefinitionRepository extends MongoRepository<RuntimeQualityAnalysisDefinition, String> { }

File: src/main/kotlin/io/github/dqualizer/dqapi/repositories/rqa/RqaDefinitionRepository.kt
package io.github.dqualizer.dqapi.repositories.rqa

import io.github.dqualizer.dqlang.types.rqa.definition.RuntimeQualityAnalysisDefinition
import org.springframework.data.mongodb.repository.MongoRepository
import org.springframework.stereotype.Repository

@Repository
interface RqaDefinitionRepository : MongoRepository<RuntimeQualityAnalysisDefinition, String>

